tools/stern/src/Stern/Types.hs

```diff
@@ -126,9 +140,45 @@ newtype MarketoResult = MarketoResult
   }
   deriving (Eq, Show, ToJSON, FromJSON)
 
+data ConsentLogAndMarketo = ConsentLogAndMarketo
+  { clamConsentLog :: ConsentLog,
+    clamMarketo :: MarketoResult
+  }
+  deriving (Eq, Show)
+
+deriveJSON toJSONFieldName ''ConsentLogAndMarketo
```

Comment:
no, because i don't want to make write schema-profunctor code that is completely dynamically typed, for stuff that contains `Value`.  and it's really not important now.  this has worked since forever, and we are in the of fixing something else.  one thing at a time.